handler/handlers.go

```diff
@@ -0,0 +1,38 @@
+
+package handler
+
+import (
+	"encoding/json"
+	"github.com/abidkhan03/go_training/csv"
+	"net/http"
+)
+
+func CsvHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Rename this file to `csv.go`
And handler to `Csv`, so it can be used as `handlers.Csv`
```suggestion
func Csv(w http.ResponseWriter, r *http.Request) {
```